hexa/plugins/connector_accessmod/graphql/schema.graphql

```diff
@@ -154,3 +148,74 @@ extend type Mutation {
     createAccessmodFile(input: CreateAccessmodFileInput): CreateAccessmodFileResult
     deleteAccessmodFile(input: DeleteAccessmodFileInput): DeleteAccessmodFileResult
 }
+
+# Analysis
+enum AccessmodAnalysisStatus {
+    PENDING
+    READY
+    QUEUED
+    RUNNING
+    SUCCESS
+    FAILED
+}
+interface AccessmodAnalysis {
```

Comment:
I'd add `errors: [AccessmodAnalyisError!]` where `AccessmodAnalysisError is something like

```graphql
enum AccessmodAnalysisError = MISSING_LAND_COVER | ... 
```

Reply:
Yup, no error handling at all ATM. Will try to add basic error handling asap.
I would go for an array of `AccessModError` (with `field`, `code` and `message` - let's discuss this)